### PR TITLE
Update Oracle user creation script

### PR DESCRIPTION
**Pull request by prees1**
- Improve note style at top of doc
- Update Oracle user creation script
- Also adds the CDB user creation script needed for Oracle CDB database support.

Still pending for another PR is to document how to configure the YAML file to support CDB database access.

### CODE REVIEW

File: oracle_cdb_user.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE USER c##tidal_comm_user IDENTIFIED BY "replace_this_with_secure_password" account unlock;
```
Comment (justinbarclay) [inline oracle_cdb_user.sql:1]: What does this commend do? It's at least highlighted weirdly?

File: pages/tidal tools/analyze_database.md [hunk 2]
```diff
@@ -57,18 +57,11 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
   - password - A password for the corresponding user.
   - name - A common name for your database could be the same or different from db_name, but this value is arbitrary and only for your reference.
 
-### Oracle User
+### Oracle User Access
 
-You can either use the default `SYSDBA` user or you can create a user with the same set of permissions:
+You can can create a user with the [script and set of permissions defined in this script](/oracle_user.sql). You should provide a secure password on the first line.
```
Comment (justinbarclay) [inline pages/tidal tools/analyze_database.md:62]: I don't know Oracle SQL too much but this looks good to me :)

**Review by justinbarclay — APPROVED** [review]

File: oracle_cdb_user.sql (hunk 1 above)
Comment (prees1) [reply oracle_cdb_user.sql:1]: not a comment. broken syntax highlightling on GitHub. `c##tidal_comm_user` is the username. and `c##` is required Oracle syntax.

File: pages/tidal tools/analyze_database.md (hunk 2 above)
Comment (prees1) [reply pages/tidal tools/analyze_database.md:62]: Thanks! can confirm, the script works.

**Pull request merged**